roxctl/generate/netpol/cta.go

```diff
@@ -0,0 +1,23 @@
+package netpol
+
+import (
+	"github.com/np-guard/cluster-topology-analyzer/pkg/controller"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/protoconv/networkpolicy"
+)
+
+func (cmd *generateNetpolCommand) generateNetpol() error {
```

Comment:
Could you add a test?

Reply:
Yes, absolutely. We will add tests in the follow up [ROX-11774](https://issues.redhat.com/browse/ROX-11774).
This branch started as a PoC and I treated the attached Epic as an opportunity to slice the work into multiple smaller issues.
This is an exception and won't be the norm, therefore I am asking for a one-time allowance to introduce tests in a second PR :)